commands/object-encoding.md

```diff
@@ -7,6 +7,7 @@ Redis objects can be encoded in different ways:
 * Sets can be encoded as `intset` or `hashtable`. The `intset` is a special encoding used for small sets composed solely of integers.
 * Hashes can be encoded as `ziplist` or `hashtable`. The `ziplist` is a special encoding used for small hashes.
 * Sorted Sets can be encoded as `ziplist` or `skiplist` format. As for the List type small sorted sets can be specially encoded using `ziplist`, while the `skiplist` encoding is the one that works with sorted sets of any size.
+* An embedded string, `embstr`, is an object where the internal simple dynamic string, `sds`, is an unmodifiable string allocated in the same chuck as the object itself. `embstr` can be strings with lengths up to the hardcoded limit of `OBJ_ENCODING_EMBSTR_SIZE_LIMIT` or `44` bytes. If the size of the value string is less than or equal to `44` bytes, Redis saves the string in the same chunk of the Redis object itself.
```

Comment:
This is en encoding of strings, so it belongs with the string type above, where the raw and int encodings are described. We can turn the string encodings into a sublist with bullets for raw, embstr and int. WDYT?

The last sentense seems redundant in some way. Also, I'm not sure Redis always uses the embstr. Some commands, for example APPEND, always stores the string as raw, even if it's short. So I guess it's better to remove the last sentence.

Btw, use a new line for each sentence. It has no effect since will not be rendered as a newline on the website, but it makes the diffs smaller later when markdown is edited.

Reply:
Resubmitted, @zuiderkwast. Please review when you get the chance.